app/build.gradle

```diff
@@ -14,6 +14,14 @@ android {
     namespace 'com.withpersona.sdk2.demo'
     compileSdk 33
 
+    // https://kotl.in/gradle/jvm/toolchain
+    // Whenever a stable AGP > 8.1.0-alpha09 is out
+    // this `compileOptions` block can be removed.
+    compileOptions {
```

Comment:
You probably also want to bump this for kotlin
```
  kotlin {
    jvmToolchain(17)
  }
```

Reply:
I tried setting up that block but couldn't get it going in the time box I set for myself. 😢